infrastructure/iac/aws-cdk/backend/rootski_backend_cdk/database/lightsail/stacks/resources/user-data.template.sh

```diff
@@ -128,25 +111,23 @@ EOF
 cd /home/ec2-user
 [[ -d /home/ec2-user/rootski ]] \
     || GIT_SSH_COMMAND='ssh -F /home/ec2-user/.ssh/config' \
-    git clone git@bitbucket.org:eriddoch1/rootski.git
+    git clone --depth 1 git@github.com:rootski-io/rootski.git
 
-# pull the latest code from the rootski repo
-cd /home/ec2-user/rootski
-git remote set-url origin git@bitbucket.org:eriddoch1/rootski.git # make sure we pull over ssh
-git stash
-GIT_SSH_COMMAND='ssh -F /home/ec2-user/.ssh/config' \
-    git pull origin
+# deploy docker stack
+cd rootski
 
-# pull the larger CSV files to seed the database; TODO - remove this in favor of restoring from S3 backup
-GIT_SSH_COMMAND='ssh -F /home/ec2-user/.ssh/config' \
-    git lfs pull
+# create directories that are mounted as volumes in the docker
+# containers but aren't in github
+sudo mkdir infrastructure/containers/postgres/data
+sudo mkdir infrastructure/containers/postgres/backups
 
-# deploy docker stack
+# install necessary dependencies, builds the docker images, and
+# runs the postgres and database-backup containers which creates
+# the database, restores it from the most recent S3 backup, and
+# sets the database to backup to S3 continually on an interval
 make install
 make build-images
-make start-backend
-make await-db-healthy
-make seed-dev-db
+make start-database-stack
```

Comment:
Ah, you know what? Your `startup.sh` script is _awesome_, but it couples `make start-database-stack` with restoring the database as well. Also, shouldn't this be `make start-database-stack-lightsail`?

Can we separate `start`ing the stack from `restore`ing the database? One issue with coupling the start to restoring is that none of the contributors who don't have access to our production database backups bucket will be able to use this makefile target. That will actually be the majority of contributors since the data is the sensitive part.

Reply:
What is the best way to do this. I was thnking once we get a mini dataset, we can separate these out. And would it be better to have separate targets that run different code or  separate targets that have identical code but PROD.env vs DEV.env files.

Reply:
To achieve this, we might need to have `make restore-from-latest-backup` use `docker-compose exec -e POSTGRES_HOST=<localhost on linux | host.docker.internal on mac> <name or ID of running backup/restore container>`.

The reason for this is that `docker-compose run ...` commands can't connect to docker swarm networks (at least I think they can't, maybe adding `attachable: true` like you did changes that`. This is roughly the way I used to have the `make restore` command working.